srcpkgs/mpvpaper/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'mpvpaper'
+pkgname=mpvpaper
+version=1.3
+revision=1
+build_style=meson
+hostmakedepends="pkg-config wayland-devel"
+makedepends="wayland-protocols wlroots-devel mpv-devel"
+depends="wayland-protocols"
+short_desc="Video wallpaper program for wlroots based wayland compositors"
+maintainer="Erick Vilcica <erick.vilcica@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/GhostNaN/mpvpaper"
+changelog="https://github.com/GhostNaN/mpvpaper/releases/tag/${version}"
```

Comment:
It's better to use the full releases page here instead, because sometimes you'll want to see the changes across multiple versions:
```suggestion
changelog="https://github.com/GhostNaN/mpvpaper/releases"
```

Reply:
Fixed.